src/agent/src/config.rs

```diff
@@ -30,6 +30,7 @@ const CONTAINER_POLICY_FILE: &str = "agent.container_policy_file";
 const HTTPS_PROXY: &str = "agent.https_proxy";
 const NO_PROXY: &str = "agent.no_proxy";
 const ENABLE_DATA_INTEGRITY: &str = "agent.data_integrity";
+const AA_KBC_PARAMS: &str = "agent.aa_kbc_params";
```

Comment:
Nit: It might be worth considering moving this line above `const HTTPS_PROXY: &str = "agent.https_proxy";` so that it is consistent with the ordering in the existing code that reads this from the agent config toml file e.g.:
```
#[derive(Debug)]
pub struct AgentConfig {
    pub debug_console: bool,
...
    pub container_policy_path: String,
    pub aa_kbc_params: String,
    pub https_proxy: String,
...
}
```

This applies to most of the changes here, but I didn't want to spam this comment in every line!

Reply:
Thanks @stevenhorsman , make sense for me, fixed now :-)